scripts/build_and_push_all.sh

```diff
@@ -1,26 +1,23 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 # exit as soon as any command fails
 set -e
 
 REPO_SCOPE=${REPO_SCOPE:-aldavidson}
 TAG=${TAG:-latest}
-AWS_ACCESS_KEY_ID_BASE=${AWS_ACCESS_KEY_ID_BASE:-${AWS_ACCESS_KEY_ID}}
-AWS_SECRET_ACCESS_KEY_BASE=${AWS_SECRET_ACCESS_KEY_BASE:-${AWS_SECRET_ACCESS_KEY}}
-AWS_ACCESS_KEY_ID_API=${AWS_ACCESS_KEY_ID_API:-${AWS_ACCESS_KEY_ID}}
-AWS_SECRET_ACCESS_KEY_API=${AWS_SECRET_ACCESS_KEY_API:-${AWS_SECRET_ACCESS_KEY}}
-AWS_ACCESS_KEY_ID_WORKER=${AWS_ACCESS_KEY_ID_WORKER:-${AWS_ACCESS_KEY_ID}}
-AWS_SECRET_ACCESS_KEY_WORKER=${AWS_SECRET_ACCESS_KEY_WORKER:-${AWS_SECRET_ACCESS_KEY}}
-
-concat_and_uppercase() {
-  echo "$1_$2" | tr '[:lower:]' '[:upper:]'
-}
 
 login_to_ecr_with_creds_for() {
-  KEY_VAR_NAME=$(concat_and_uppercase "AWS_ACCESS_KEY_ID" $1)
-  SECRET_VAR_NAME=$(concat_and_uppercase "AWS_SECRET_ACCESS_KEY" $1)
-  echo "Logging in with per-repo credentials ${KEY_VAR_NAME} ${SECRET_VAR_NAME}"
-  export AWS_ACCESS_KEY_ID=${!KEY_VAR_NAME}
-  export AWS_SECRET_ACCESS_KEY=${!SECRET_VAR_NAME}
+  AWS_ACCESS_KEY_ID_ENCODED=`kubectl -n formbuilder-repos get secrets ecr-repo-fb-submitter-$TYPE -o jsonpath='{.data.access_key_id}'`
+  AWS_SECRET_ACCESS_KEY_ENCODED=`kubectl -n formbuilder-repos get secrets ecr-repo-fb-submitter-$TYPE -o jsonpath='{.data.secret_access_key}'`
+
+  if [ `uname` == 'Darwin' ]
+  then
+    export AWS_ACCESS_KEY_ID=`echo $AWS_ACCESS_KEY_ID_ENCODED | base64 --decode`
+    export AWS_SECRET_ACCESS_KEY=`echo $AWS_SECRET_ACCESS_KEY_ENCODED | base64 --decode`
+  else
+    export AWS_ACCESS_KEY_ID=`echo $AWS_ACCESS_KEY_ID_ENCODED | base64 -d`
```

Comment:
Any reason we prefer shorthand decode flag when not on Darwin?
If both used --decode then we could get rid of this conditional?

Reply:
unfortunately the api is different on mac and alpine

mac:
```
Usage:	base64 [-hvD] [-b num] [-i in_file] [-o out_file]
  -h, --help     display this message
  -D, --decode   decodes input
  -b, --break    break encoded string into num character lines
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
```

alpine (fb-builder):
```
/ # echo "foo" | base64 --decode
base64: unrecognized option: decode
BusyBox v1.29.3 (2019-01-24 07:45:07 UTC) multi-call binary.

Usage: base64 [-d] [FILE]

Base64 encode or decode FILE to standard output
	-d	Decode data
```

Reply:
Oh I see, thanks.